docs/guides/server/containers.adoc

```diff
@@ -229,4 +229,26 @@ podman|docker run --name keycloak_unoptimized -p 8080:8080 \
 
 Feel free to join the open https://github.com/keycloak/keycloak/discussions/8549[GitHub Discussion] around enhancements of the admin bootstrapping process.
 
+== Specifying different memory settings
+
+The {project_name} container, instead of specifying hardcoded values for the initial and maximum heap size, uses relative values to the total memory of a node.
+This behavior is achieved by JVM options `-XX:MaxRAMPercentage=70`, and `-XX:InitialRAMPercentage=50`.
+
+The `-XX:MaxRAMPercentage` option represents the maximum heap size as 70% of total node memory.
+The `-XX:InitialRAMPercentage` option represents the initial heap size as 50% of the total node memory.
```

Comment:
This mentions "node", where it should probably mention "container"? The node's memory is not relevant to the JVM IMHO.

Reply:
@ahus1 You're right, thanks